### ISSUE TITLE
Shawburn: When setting Primary colour in Customizer, button hover turns white

### ISSUE THREAD

**Issue by kathrynwp** [Automattic/wp-calypso#60109]
### Quick summary

When setting a Primary colour in Shawburn in the Customizer, hovering over any buttons turns the background white

### Steps to reproduce

1. Activate Shawburn on a Simple or Atomic site
2. Go to Appearance > Customize > Colors & Backgrunds
3. Choose a colour under Primary:
![Customize__Global_Styles_Demo_–_An_awesome_tagline](https://user-images.githubusercontent.com/7662449/149677886-b9f4799a-f769-441f-8841-da3e4a76a82c.jpg)
4. Add a button block to a page or post, or add a block with a button included in it, such as the Search block

### What you expected to happen

The button should turn a lighter version of the selected colour

### What actually happened

The button turns white, i.e. invisible against a white background.

Note that the issue occurs both **in the editor and on the front end**

### Context

WordPress.com customer chat: 31438790-hc

### Simple, Atomic or both?

Simple, Atomic

### Theme-specific issue?

Shawburn (may also affect other themes)

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

Yes, easy to implement

### Workaround details

To work around the issue, select the block and manually set the background colour.

![Edit_Page_“Home”_‹_Global_Styles_Demo_—_WordPress](https://user-images.githubusercontent.com/7662449/149678006-e1964181-e4b5-4e91-8623-75db5f0d529f.jpg)

Note that while this is an "easy" workaround in theory once we show someone how to do it, it is not very discoverable or intuitive and users are likely to need assistance in figuring it out. The other limitation is that this sets the hover colour to the same solid colour as the background, instead of making it a lighter shade, which is how the buttons are supposed to behave.

### Screencast:

https://user-images.githubusercontent.com/7662449/149678159-40d36a8b-0495-4042-b8ab-9328aba4fbc1.mov

Custom CSS is also possible as a front-end fix, but since this bug _also_ affects the editor side, custom CSS is only a partial workaround, and of course doesn't help users who don't have access to Custom CSS.

**Comment by kerrynicl** [comment]
@kathrynwp Hi There, I'm not able to replicate this on either a simple or AT site. Could you confirm if this is still happens on your end?
![Screen Capture on 2022-01-17 at 08-22-44 (1)](https://user-images.githubusercontent.com/83325600/149777156-6cd425fc-b40d-470f-9c1d-b1999a459802.gif)

![Screen Capture on 2022-01-17 at 08-16-28 (1)](https://user-images.githubusercontent.com/83325600/149776173-28460c3c-1ef2-4c04-8157-d8eef3ba6d4e.gif)

**Comment by kathrynwp** [comment]
@kerrynicl Yes, it's still happening. Please check out the "Portfolio" button on my test site here: P2xeYC-1th-p2 (will try to keep this running Shawburn until you can take a look!)
